src/nyc_trees/apps/survey/models.py

```diff
@@ -13,6 +13,18 @@
 from libs.mixins import NycModel
 
 
+class Borough(NycModel, models.Model):
+    geom = models.MultiPolygonField()
+    name = models.CharField(max_length=32)
+    code = models.IntegerField(db_index=True, unique=True)
```

Comment:
We don't use ``code`` in any sorts or joins (We did in an early draft of the feature) so we can drop these indexes.